terraform/test-environments/scenarios/single_local_inplace_upgrade.tf

```diff
@@ -18,7 +18,7 @@ module "single_local_inplace_upgrade" {
   vpc            = "${var.environment}"
   platform       = "ubuntu-16.04"
   key_name       = "cd-infrastructure"
-  instance_type  = "m4.large"
+  instance_type  = "m5.large"
```

Comment:
does this mean we will lose the data that is currently on these instances?

Reply:
No, I already resized them to m5.large this morning during the maintenance window.  During the update I just restarted the instance, so you did not loose any data